content/setup-ubuntu-linux-for-laravel-developer.md

```diff
@@ -0,0 +1,111 @@
+php composer-setup.php
+php -r "unlink('composer-setup.php');"
+```
+
+Setelah proses diatas sudah dijalankan satu-persatu, pindahkan composer.phar ke direktori PATH.
+
+```bash
+sudo mv composer.phar /usr/local/bin/composer
+```
+
+# NodeJs
+
+Kita perlu memasang NodeJS untuk memasang NPM (Node Package manager), caranya pun cukup mudah. Jalankan perintah dibawah ini satu per satu:
+
+```bash
+curl -sL https://deb.nodesource.com/setup_17.x -o nodesource_setup.sh
+sudo bash nodesource_setup.sh
+sudo apt install nodejs
+```
+
+Untuk version pada `https://deb.nodesource.com/setup_17.x`, ini opsional ya. kalian bisa menyesuaikan, sesuai dengan kebutuhan.
+
+# MySQL
+
+Nah, untuk MySQL ini kalian sebenarnya tidak perlu bingung. Cukup buka link ini
+
+- [`Mysql Download`](https://dev.mysql.com/downloads/mysql/),
+
+Download MySQL-nya dan sesuaikan dengan OS Distro Linux kalian. Kemudian ikuti langkah-langkahnya seperti yang ada di dokumentasi ini
+
+- [`Cara Install`](https://dev.mysql.com/doc/refman/8.0/en/linux-installation-debian.html).
+
+Lohhh, kok ke dokumentasi? Ya sekalian belajar baca-baca dokumentasinya juga hehehe :). Belajar dari yang mudah untuk diikuti terlebih dahulu aja okay ^~^.
+
+Untuk MySQL Client nya sendiri, saya menggunakan ekstensi Database yang ada di Visual Studio Code. Jadi, saya ga perlu install PHPMyAdmin, MySQL Client, atau sejenisnya.
+
+Oiya, disini saya tidak menggunakan Web Server ya. Karena Laravel sendiri sudah menyediakannya dengan cara menjalankan command `php artisan serve`.
+
+Sip, selanjutnya kita install Go untuk kebutuhan kalau kita pengen ngoding Go.
+
+# Go
+
+Sebenarnya, langkahnya cukup mudah. Pertama [Download Go disini](https://go.dev/doc/install), setelah itu jalankan perintah dibawah ini satu per satu
+
+```bash
+rm -rf /usr/local/go && tar -C /usr/local -xzf go1.17.6.linux-amd64.tar.gz
+export PATH=$PATH:/usr/local/go/bin
+```
+
+Udah deh, tinggal cek. Apakah sudah terpasang atau belum dengan command
+
+```bash
+go version
+```
+
+# PosgreSQL
+
+Yapss, karena kebanyakan Go Developer lebih suka menggunakan PostgreSQL dari pada MySQL. Maka kita akan install juga, caranya ini cukup mudah. Cukup ikuti langkah-langkahnya seperti yang ada di dokumentasi resmi ini
+
+- [Cara Install PostgreSQL](https://www.postgresql.org/download/linux/ubuntu/)
+
+Kalau sudah, untuk PostgreSQL Clientnya sendiri, saya juga menggunakan ekstensi yang ada di Visual Studio Code / Goland.
+
+`kalian : boleh minta link ekstensinya ga om?`
+
+`saya : boleh, tapi enggak dibahas disini ya, 😁😁😁.`
+
+Kenapa beberapa langkah saya arahkan ke dokumentasi langsung? ya sambil belajar baca dokumentasi dikit-dikitlah ya, supaya enggak terjebak di tutorial hell mhehehe :)
+
+Okay cukup sekian,
+
+What's next? if you have some req tips. reach me on twitter [@a17wahidin](https://twitter.com/a17wahidin)
+
+```bash
+PHP IS STILL ALIVE.
+BUT, SOMETIME GO IS MAKE ME HAPPY TO CODE 😆.
```

Comment:
fixed: "BUT GO IS WHAT MAKES ME HAPPY TO CODE"

Reply:
masih belum diperbaiki

Reply:
hooo ma bad. HUMAN error :/